sudoku/validate_sudoku.py

```diff
@@ -0,0 +1,119 @@
+# validate sudoku
+# input is a board
+# Return True if board is correct
+# Return False if board is incorrect
+
+
+def validate(board):
+
+    board_len = len(board)
+
+    # # check row numbers for duplicates
+    seen = {}
+    dups = []
+    for row in range(board_len):
```

Comment:
Don't need range here. The code can be a lot simpler:
```suggestion
    for row in board:
        seen = set()
        for val in row:
            if val in seen:
                # Invalid
                return False
             
             seen.add(val)
```

Reply:
fixed the code